packages/app/src/components/nominationInfo/index.js

```diff
@@ -1,20 +1,33 @@
 import React, { useState, useContext, useEffect } from 'react';
 import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
 import ApplicationDetail from "./applicationDetail";
+import HealthProviderDetail from "./healthProviderDetail";
 import styles from "./styles.module.css";
 
 
+
+
+
+
 function NominationInfo() {
   const [activeNomination, setActiveNomination] = useContext(
     ActiveNominationContext
   );
 
+  
```

Comment:
Can you remove this extra whitespace please?

Reply:
Will remove right now!